core/src/main/scala/chandu0101/scalajs/react/components/ReactTable.scala

```diff
@@ -10,18 +10,22 @@ import scalacss.ScalaCssReact._
 
 object ReactTable {
 
-  type Model = Map[String, Any]
+  type ColumnKey = String
+  type Model = Map[ColumnKey, Any]
```

Comment:
I actually did this when i tried to use `ReactTable` for a project, so it works. the only thing you need to specify for a given type `T` is a function `T => String` or something similar, it might have been for the rendering i needed it - dont remember well.